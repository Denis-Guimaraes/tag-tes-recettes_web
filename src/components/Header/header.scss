@import 'src/styles/vars';

.header {
  padding: $base-padding*2;
  background-color: $color-pink;
  text-transform: uppercase;

  &__title {
    font-family: $fantasy-font;
    font-size: $base-font-size*2;
  }
}
