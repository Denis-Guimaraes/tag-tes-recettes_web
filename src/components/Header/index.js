// NPM import
import React from 'react';
import { Link } from 'react-router-dom';

// Local import

// Styles
import './header.scss';

// Code
const Header = () => (
  <header className="header">
    <h1 className="header__title">
      <Link to="/">
        Tag Tes Recettes
      </Link>
    </h1>
  </header>
);

// Export
export default Header;
