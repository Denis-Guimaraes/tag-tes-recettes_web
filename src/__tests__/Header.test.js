/* eslint-disable no-undef */
// NPM import
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Link } from 'react-router-dom';
// Local import
import Header from '../components/Header';


describe('Header', () => {
  it('match snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('contain component Link', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.exists(Link)).toBe(true);
  });
});
