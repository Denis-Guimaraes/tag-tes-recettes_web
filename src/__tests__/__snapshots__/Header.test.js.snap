// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header match snapshot 1`] = `
<header
  className="header"
>
  <h1
    className="header__title"
  >
    <Link
      replace={false}
      to="/"
    >
      Tag Tes Recettes
    </Link>
  </h1>
</header>
`;
